adserver/views.py

```diff
@@ -798,11 +798,17 @@ def get_context_data(self, **kwargs):
         advertiser_slug = kwargs.get("advertiser_slug", "")
         advertiser = get_object_or_404(Advertiser, slug=advertiser_slug)
 
+        flight_slug = self.request.GET.get("flight", "")
+        flight = Flight.objects.filter(
+            campaign__advertiser=advertiser, slug=flight_slug
+        ).first()
```

Comment:
Does the rest of the code handle `flight=None` in a good way? Should we error on this? Would only happen w/ URL hacking I guess, so probably not a huge deal.

Reply:
It handles `flight=None` as not filtering on flights. It will display results for all flights.